presto-product-tests/conf/docker/singlenode-mapr/compose.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+source ${BASH_SOURCE%/*}/../common/compose-commons.sh
+
+docker-compose \
+-f ${BASH_SOURCE%/*}/../common/standard.yml \
+-f ${BASH_SOURCE%/*}/../common/jdbc_db.yml \
+-f ${BASH_SOURCE%/*}/../common/cassandra.yml \
```

Comment:
https://github.com/prestodb/presto/pull/7684 is in master already, but I understand this PR's base does not contain it? I'd prefer we got rid of that before having this in sprint branch